ROADMAP.md

```diff
@@ -8,59 +8,65 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 1.2
 
-* Integration Tests
-  * Speed up integration tests [#1218](https://github.com/rook/rook/issues/1218)
+## Rook 1.4
+
+The following features are targeted for Rook v1.4 (July 2020).
```

Comment:
Additionally, we will work on:

* https://github.com/rook/rook/issues/5356
* https://github.com/rook/rook/issues/5516

And we might be able to do this one: https://github.com/rook/rook/issues/5006.

Please add them in the list. Thanks!

Reply:
I'd like to keep the features in the roadmap at a high level. I'll add a link to the project board where we will track all the issues being fixed. Sound ok to keep these in the project board?

Reply:
They already are :) but perhaps we can mention "external cluster enhancements"?